File: LunchApp/app/src/main/java/br/com/gersonsilvafilho/lunchapp/data/VotesServiceApi.java
package br.com.gersonsilvafilho.lunchapp.data;

import java.util.List;
import java.util.Map;

import retrofit.Call;
import retrofit.http.GET;
import retrofit.http.Query;

/**
 * Created by GersonSilva on 12/22/16.
 */

public interface VotesServiceApi
{
    @GET("/vote.php")
    Call<Map<String,String>> SendVote(@Query("dateTime") long unixtime,
                        @Query("restaurantId") String restaurantId,
                        @Query("userId") String userid);

    @GET("/vote.php")
    Call<Map<String,String>> ChangeVote(@Query("dateTime") long unixtime,
                          @Query("restaurantId") String restaurantId,
                          @Query("userId") String userid);

    @GET("/list.php?")
    Call<Map<String,List<String>>> GetVotesFromDay(@Query("dateTime") long unixtime);
}

File: LunchApp/app/src/main/java/br/com/gersonsilvafilho/lunchapp/data/VotesServiceApi.kt
package br.com.gersonsilvafilho.lunchapp.data

import retrofit.Call
import retrofit.http.GET
import retrofit.http.Query

/**
 * Created by GersonSilva on 12/22/16.
 */

interface VotesServiceApi {
    @GET("/vote.php")
    fun SendVote(@Query("dateTime") unixtime: Long,
                 @Query("restaurantId") restaurantId: String,
                 @Query("userId") userid: String): Call<Map<String, String>>

    @GET("/vote.php")
    fun ChangeVote(@Query("dateTime") unixtime: Long,
                   @Query("restaurantId") restaurantId: String,
                   @Query("userId") userid: String): Call<Map<String, String>>

    @GET("/list.php?")
    fun GetVotesFromDay(@Query("dateTime") unixtime: Long): Call<Map<String, List<String>>>
}
